src/baseevents.cpp

```diff
@@ -61,15 +61,19 @@ bool BaseEvents::loadFromXml()
 			continue;
 		}
 
+		bool success;
+
 		pugi::xml_attribute scriptAttribute = node.attribute("script");
 		if (scriptAttribute) {
 			std::string scriptFile = "scripts/" + std::string(scriptAttribute.as_string());
-			event->checkScript(basePath, scriptsName, scriptFile) && event->loadScript(basePath + scriptFile);
+			success = event->checkScript(basePath, scriptsName, scriptFile) && event->loadScript(basePath + scriptFile);
 		} else {
-			event->loadFunction(node.attribute("function"));
+			success = event->loadFunction(node.attribute("function"));
```

Comment:
Functions are generally being removed, it would be better to wait until they are removed so you won't need this book :)